spec/index.bs

```diff
@@ -8469,6 +8469,13 @@ dictionary GPURenderPipelineDescriptor
         - The [=location=] of each user-defined output of
             |descriptor|.{{GPURenderPipelineDescriptor/vertex}} must be
             &le; |maxVertexShaderOutputLocation|.
+    1. If |device|.{{device/[[features]]}} does not [=list/contain=] {{GPUFeatureName/"core-features-and-limits"}}:
+            <div class="compatmode">
+        - For each user-defined |output| of |descriptor|.{{GPURenderPipelineDescriptor/vertex}}:
+            - If the [=interpolation=] of the |output| is [=interpolation type/linear=], return `false`.
+            - If the [=interpolation=] of the |output| is [=interpolation type/flat=] and the [=interpolation sampling=] is not [=interpolation sampling/either=], return `false`.
+            - If the [=interpolation sampling=] of the |output| is [=interpolation sampling/sample=] or [=interpolation sampling/first=], return `false`.
```

Comment:
nit: Rules about "either" and "first" on these two lines seem redundant?

Reply:
Good point. I wanted to prevent pathological cases like "linear, first", but it looks like that's covered by the WGSL spec already.

Removing the "either" condition from the first line works but seems a bit unintuitive. I've removed the "first" condition from the second line instead.